vorestation.dme

```diff
@@ -1641,9 +1641,9 @@
 #include "code\modules\ai\ai_holder_targeting_vr.dm"
 #include "code\modules\ai\interfaces.dm"
 #include "code\modules\ai\say_list.dm"
-#include "code\modules\ai\ai_holder_subtypes\simple_mob_ai.dm"
-#include "code\modules\ai\ai_holder_subtypes\simple_mob_ai_vr.dm"
-#include "code\modules\ai\ai_holder_subtypes\slime_xenobio_ai.dm"
+#include "code\modules\ai\aI_holder_subtypes\simple_mob_ai.dm"
+#include "code\modules\ai\aI_holder_subtypes\simple_mob_ai_vr.dm"
+#include "code\modules\ai\aI_holder_subtypes\slime_xenobio_ai.dm"

```

Comment:
I assume this wasn't intentional, please discard these.

Reply:
Problem is the folder is actually capitalized here on the repo as well (and upstream at Polaris). The folder name needs fixing or else this is going to persist.

Reply:
There was a fix merged a while ago but it got undone hahaha

Reply:
Reminder that git does not actually store 'folders'. It doesn't even really store files, just commits that it can use to rebuild the files. But it does store the paths to those files. There are 3 files in that folder that had the wrong path stored.

https://github.com/VOREStation/VOREStation/pull/8303

Reply:
Oh, let me get that. I have no idea why it changed that, since I never modified those files.